arrange_schedule.py

```diff
@@ -1120,6 +1120,14 @@ def expire_data_check():
         receive_obj["error"] = "expire_data_check : {errorMsg}".format(errorMsg=receive_obj["error"])
         set_system_log(receive_obj)
 
+def do_google_calendar():
+    receive_obj = google_calendar_text()
+    if receive_obj["result"] == "success":
+        "DO NOTHING"
+    else:
+        receive_obj["error"] = "google_calendar_text" + receive_obj["error"]
+        set_system_log(receive_obj)
+
```

Comment:
Please add try....catch in google_calendar_text() [line723], or some exception log will not write to set_system_log.

`def google_calendar_text():`
`try:`
[line723~740]
`except Exception as e:`
`return_msg["error"] = str(e)`
`return return_msg`